ui/raidboss/data/06-ew/raid/p4s.txt

```diff
@@ -75,12 +75,12 @@ hideall "--sync--"
 # Towers
 250.1 "Belone Coils" sync / 1[56]:[^:]*:Hesperos:69DD:/
 260.3 "Inversive Chlamys" sync / 1[56]:[^:]*:Hesperos:69ED:/
-260.9 "Burst" sync / 1[56]:[^:]*:Hesperos:(69DE|69DF):/
+260.9 "Burst" sync / 1[56]:[^:]*:Hesperos:(69DE|69DF|69E0|69E1):/
```

Comment:
Could you pull all the timeline changes out into its own PR? I think I could land that immediately.

Reply:
I'll be able to later.